@lingosta/app/providers/useUser.tsx

```diff
@@ -0,0 +1,87 @@
+"use client";
```

Comment:
let's name this file `UserProvider` instead of `userUser`. `useUser` is the hook inside of this.